Tests/bounding-rectangles.lisp

```diff
@@ -0,0 +1,115 @@
+(in-package #:clim-tests)
+
+;;; TODO
+;;;
+;;; - add tests for the standard region sets (union, difference,
+;;;   intersection and rectangle-set). Sets should be tested for
+;;;   elements of different dimensions (points, paths and areas).
+;;; - add tests for unbounded regions (should signal error).
+;;; - add tests which verify correctness of a bounding rectangle.
+;;; - add tests for rotated regions
+;;; - randomize coordinates (translation and scaling affect the
+;;;   bounding-rectangle in a predictable way)
+;;; - test other instationable bounding-rectangle subclasses
+
+
+(def-suite* :mcclim.bounding-rectangles
+  :in :mcclim)
+
+(test bounding-rectangle.point
+  (let* ((point (make-point 0 0))
+         (point-br (bounding-rectangle point)))
+    (is (bounding-rectangle-p point-br))
+    (with-bounding-rectangle* (x1 y1 x2 y2) point
+      (is (and (coordinate= 0 x1)
+               (coordinate= 0 x2)
+               (coordinate= 0 y1)
+               (coordinate= 0 y2))))))
```

Comment:
Please write these as
```lisp
(is (coordinate= 0 x1))
(is (coordinate= 0 x2))
…
```
This style allows fiveam to provide a more detailed description in case one of the assertions fails.

This happened in my test run and I just got:
```
MCCLIM.BOUNDING-RECTANGLES.BOUNDING-RECTANGLE.ELLIPTICAL-ARC
Error Details
(AND (COORDINATE= 0 X1) (COORDINATE= -15 Y1) (COORDINATE= 10 X2) (COORDINATE= 0 Y2)) was NIL.
```

Reply:
Maybe even better, make a helper function that asserts `bounding-rectangle-p` and the four values.